packages/kitchen/src/components/Link/index.tsx

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import styled from "styled-components";
+import { KitchenComponent } from "../../types";
+
+export type LinkProps = KitchenComponent & {
+  /**
+   * The link's href.
+   */
+  href?: string;
+
+  /**
+   * The link's id.
+   */
+  id?: string;
+
+  /**
+   * The link's classname.
+   */
+  className?: string;
+
+  /**
+   * The link's onClick.
+   */
+  onClick?: (event: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => void;
+
+  /**
+   * The link's status (if it's disabled or not).
+   */
+  disabled?: boolean;
+
+  /**
+   * The link's variant.
+   */
+  variant?: "highlight" | "primary" | "secondary" | "blend";
+};
+
+const Link = styled(
+  ({
+    as: Component = "a",
+    href,
+    id,
+    className,
+    disabled,
+    onClick,
+    variant,
+    children,
+    ...props
+  }: LinkProps) => {
+    disabled = disabled || !onClick || !href;
+    return (
+      <Component
+        className={className}
+        id={id}
+        disabled={disabled}
+        onClick={onClick}
+        href={href}
+        variant={variant}
+        {...props}
+      >
+        {children}
+      </Component>
+    );
+  }
+)<LinkProps>`
+  box-sizing: border-box;
+  display: inline-flex;
+  align-items: center;
+  justify-content: space-between;
+  user-select: none;
+  flex: 1;
+  gap: 10px;
+  border-radius: 8px;
+  background-color: ${({ theme }) => {
+    return theme.colors.layout.dark;
+  }}
+    return "transparent";
+  }};
+
+  color: ${({ theme, variant, disabled }) => {
+    if (variant === "highlight") {
+      return theme.colors.accent.info;
+    } else if (variant === "blend") {
+      return theme.colors.accent.danger;
```

Comment:
go here: https://vercel.com/design/link#blend
- change the type of the note to "success"
- inspect the code to understand what is blend variant